hubble-be/src/main/java/com/baidu/hugegraph/entity/load/FileSetting.java

```diff
@@ -74,4 +74,19 @@
     @MergeProperty
     @JsonProperty("list_format")
     private ListFormat listFormat = new ListFormat();
+
+    public void unescapeDelimiterIfNeeded() {
+        if ("\\t".equals(this.delimiter)) {
+            this.delimiter = "\t";
+        }
+        if (!",".equals(this.delimiter)) {
+            this.format = "TEXT";
+        }
```

Comment:
why set format to TEXT

Reply:
Because loader restricts when format is CSV, the dekimiter must be a comma